### PR TITLE
GitISPF to fix https repo

### PR DESCRIPTION
**Pull request by xhouis**
Git ISPF fix to manage clone and Git commands using https repositories.

### CODE REVIEW

File: IDE/GitISPFClient/sbgzexec/bgzconf.rexx [hunk 1]
```diff
@@ -49,6 +49,7 @@
   Perl5_HOME   = '/var/rocket'
   DBB_HOME     = '/var/dbb103'
   ICONV_HOME   = '/bin/iconv'
+  CAINFO       = 'export GIT_SSL_CAINFO=/u/dohertl/Test/cacert.pem'
```
Comment (drbruce-git) [inline IDE/GitISPFClient/sbgzexec/bgzconf.rexx:52]: @xhouis why are we adding a line with a developer's directory and file?

Comment (xhouis) [reply IDE/GitISPFClient/sbgzexec/bgzconf.rexx:52]: @drbruce-git Good point, we need to change this to
 CAINFO       = 'export GIT_SSL_CAINFO=/var/rocket/cacert.pem'
Do i need to push the fix on same local branch ?

Comment (drbruce-git) [reply IDE/GitISPFClient/sbgzexec/bgzconf.rexx:52]: @xhouis  yes if you make the change on your local branch and then commit and push to your fork, it should be picked up automatically by the pull request.

Comment (xhouis) [reply IDE/GitISPFClient/sbgzexec/bgzconf.rexx]: @drbruce-github Ok done

**Pull request merged**